src/main/resources/site/parts/bkibolCalculator/bkibolCalculator.ts

```diff
@@ -50,46 +59,47 @@ function renderPart(req) {
     })
   }
 
-  const pageContributions = bkibolCalculator.component.renderPageContributions({})
+  const pageContributions: string = bkibolCalculator.component.renderPageContributions({})
   return {
     body: bkibolCalculator.body,
     pageContributions
   }
 }
 
-function getBkibolCalculatorComponent(page) {
-  const part = getComponent()
-  const language = getLanguage(page)
-  const phrases = language.phrases
-  const months = allMonths(phrases)
-  const config = getCalculatorConfig()
-  const bkibolDataEnebolig = getBkibolDatasetEnebolig(config)
-  const lastUpdated = lastPeriod(bkibolDataEnebolig)
-  const nextUpdate = nextPeriod(lastUpdated.month, lastUpdated.year)
-  const nextReleaseMonth = nextUpdate.month === 12 ? 1 : nextUpdate.month + 1
-  const nextPublishText = i18nLib.localize({
+function getBkibolCalculatorComponent(page: Content<BkibolCalculatorPartConfig>): CalculatorComponent  {
+  const part: Component<BkibolCalculatorPartConfig> = getComponent()
+  const language: Language = getLanguage(page)
+  const phrases: Phrases = language.phrases as Phrases
+  const code: string = language.code ? language.code : 'nb'
+  const months: Array<MonthPhrase> = allMonths(phrases)
```

Comment:
Could allMonths, lastPeriod, nextPeriod, monthLabel be imported from /lib/ssb/utils/calculatorUtils instead (some 
 functions may require tweeks)?